src/import/YAMLschema.ts

```diff
@@ -0,0 +1,449 @@
+        code: { type: 'string' },
+        display: { type: 'string' },
+        userSelected: { type: 'boolean' }
+      },
+      additionalProperties: false
+    },
+    codeableConcept: {
+      type: 'object',
+      properties: {
+        coding: {
+          type: 'array',
+          items: {
+            $ref: '#/definitions/coding'
+          }
+        },
+        text: { type: 'string' }
+      },
+      additionalProperties: false
+    },
+    quantity: {
+      type: 'object',
+      properties: {
+        value: { type: 'number' },
+        comparator: { type: 'string' },
+        unit: { type: 'string' },
+        system: { type: 'string' },
+        code: { type: 'string' }
+      },
+      additionalProperties: false
+    },
+    range: {
+      type: 'object',
+      properties: {
+        low: {
+          oneOf: [{ type: 'string' }, { $ref: '#/definitions/quantity' }]
+        }
+      },
+      additionalProperties: false
+    },
+    configReference: {
+      type: 'object',
+      properties: {
+        reference: { type: 'string' },
+        type: { type: 'string' },
+        identifier: { $ref: '#/definitions/identifier' },
+        display: { type: 'string' }
+      },
+      additionalProperties: false
+    },
+    identifier: {
+      type: 'object',
+      properties: {
+        use: { type: 'string' },
+        type: {
+          oneOf: [{ $ref: '#/definitions/codeableConcept' }, { type: 'string' }]
+        },
+        system: { type: 'string' },
+        value: { type: 'string' },
+        period: { $ref: '#/definitions/period' },
+        assigner: { $ref: '#/definitions/configReference' }
+      },
+      additionalProperties: false
+    },
+    jurisdiction: {
+      oneOf: [{ type: 'string' }, { $ref: '#/definitions/codeableConcept' }]
+    },
+    dependencyDetails: {
+      type: 'object',
+      properties: {
+        id: { type: 'string' },
+        uri: { type: 'string' },
+        version: { type: ['string', 'number'] },
+        reason: { type: 'string' }
+      },
+      additionalProperties: false
+    },
+    configResource: {
+      type: 'object',
+      properties: {
+        reference: {
+          type: 'object',
+          properties: {
+            reference: { type: 'string' },
+            type: { type: 'string' },
+            identifier: { $ref: '#/definitions/identifier' },
+            display: { type: 'string' }
+          },
+          additionalProperties: false
+        },
+        fhirVersion: {
+          oneOf: [{ type: 'string' }, { type: 'array', items: { type: 'string' } }]
+        },
+        name: { type: 'string' },
+        description: { type: 'string' },
+        exampleBoolean: { type: 'boolean' },
+        exampleCanonical: { type: 'string' },
+        isExample: { type: 'boolean' },
+        profile: {
+          oneOf: [{ type: 'string' }, { type: 'array', items: { type: 'string' } }]
+        },
+        groupingId: { type: 'string' }
+      },
```

Comment:
We also support `extension` on each resource, as this is how you get the special binary examples of logical models to work.